types/styled-theming/index.d.ts

```diff
@@ -1,14 +1,17 @@
 // Type definitions for styled-theming 2.2
 // Project: https://github.com/styled-components/styled-theming#readme
 // Definitions by: Arjan Jassal <https://github.com/ArjanJ>
+//                 Hieu Ho <https://github.com/hieuhlc>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.7
+// TypeScript Version: 3.4
```

Comment:
This change will require all of your package's consumers to use TS 3.4 or above.
Does this change truly make the package incompatible with earlier versions than that?

Reply:
Version 2.7 didn't work so I changed it to match my current TS version. I fixed it. It only needs to catch up with `styled-components` (version 2.9)
Thank you